pages/agent/v3/aws/secrets_manager.md

```diff
@@ -46,6 +46,9 @@ To ensure your Elastic CI Stack for AWS has access to the secret:
 }
 ```
 
+>📘 Single instance Elastic CI Stacks
+> If you have set `minSize` and `maxSize` parameters equal to 1 in your [Elastic CI Stack parameters](/docs/agent/v3/elastic-ci-aws/parameters), you will only need to set the `IAMRole` to your Secret Manager secret's resource policy as above. The  `AutoscalingLambdaExecutionRole` IAM role and corresponding autoscaling resources are not created in this Elastic CI Stack setup.
```

Comment:
I *think* this is a little simpler/clearer:

```suggestion
>📘 Single instance stacks
> If you have set `minSize` and `maxSize` [parameters](/docs/agent/v3/elastic-ci-aws/parameters) equal to `1`, don't include the `AutoscalingLambdaExecutionRole` IAM role in the resource policy. Only the `IAMRole` is required since this setup doesn't need autoscaling resources.
```

But it's a little less precise, so happy to go with your version if you think it works better. Let me know what you think.